<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1462</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Среда обитания</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createPopulation">
         <property name="text">
          <string>Создать популяцию</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Выберите день для просмотра графиков: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numberOfIterations">
           <property name="maximum">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="startNewDay">
         <property name="text">
          <string>Посмотреть графики за выбранный день</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>781</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="10" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Процент потребителей в день</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QSpinBox" name="shelfLife">
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Число людей в популяции</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QComboBox" name="newProductUserColor"/>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Срок сохранности продукта в днях</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="numberOfPeople">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Цвет вновь созданной особи</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QComboBox" name="newAgentColor">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Цвет потребителя</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QSpinBox" name="consumerPercentage">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QChartView" name="chartView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
